css/docs.css

```diff
@@ -0,0 +1,159 @@
+/* Global Variables */
+:root {
+  /* color */
+  --header-bg: #1F2937;
+  --light-grey: #F1F3F5;
```

Comment:
Your variable should specify its purpose: `--light-grey-color`.

Reply:
fixed